cob_bringup/drivers/laser_scan_filter.launch

```diff
@@ -5,11 +5,17 @@
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
 	<arg name="name"/>
 
-	<!-- scan filters-->
+	<!-- angular bounds filter-->
 	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/robots/$(arg robot)/config/$(arg name).yaml"/>
 	<node ns="$(arg name)" pkg="cob_sick_s300" type="cob_scan_filter" name="scan_filter" respawn="false" output="screen">
```

Comment:
why not using the `laser_filter` package here too, as suggested by @ipa-srd in https://github.com/ipa320/cob_robots/issues/732#issuecomment-339301180

Reply:
because of https://github.com/ipa320/cob_robots/issues/732#issuecomment-339601876